docker-compose.yml

```diff
@@ -7,6 +7,20 @@ services:
     ports:
       - 5000:5000
 
+  nztc-open-loop:
+    build:
+      context: environment
+      dockerfile: Dockerfile-open-loop
+    ports:
+      - 5001:5001      
+
+  nztc-closed-loop:
+    build:
+      context: environment
+      dockerfile: Dockerfile-closed-loop
+    ports:
+      - 5002:5002
```

Comment:
These are isolated containers, so they should all use port 5000.

Actually, we'll only need a single `nztc` container and we'll select the relevant gym environment using a shell environment variable.